dti-website/src/data/all-projects.json

```diff
@@ -267,148 +91,188 @@
     "hero": {
       "header": "What We Do",
-      "image": "",
-      "subheader": "Shout helps raise awareness of sexual assault on campus and gives survivors a voice to share their stories and locate regions on campus where incidents have occurred."
+      "image": "/static/projects/zing-header.png",
+      "subheader": "Zing is an all-in-one web interface for professors to create optimal groups for class projects based on students’ study preferences. Built with the GroupEng algorithm developed by the Director of Cornell Engineering’s MTEI program, Zing enables professors to customize grouping criteria and easily match students. Our team also supports the Learning Strategies Center’s Study Partner program as a companion tool. Together, Zing helps student succeed inside and outside the classroom."
     },
-    "ios-github": "https://github.com/cornell-dti/shout-ios",
-    "android-github": "https://github.com/cornell-dti/shout-android",
-    "heroStartingColor": "#A1F6F4",
-    "heroEndingColor": "#FAFAFA",
+    "github": "https://github.com/cornell-dti/zing",
+    "heroStartingColor": "#9D4CDB",
+    "heroEndingColor": "#F4E8FE",
     "heroUseDarkText": true,
     "features": [
       {
-        "title": "Find Out.",
-        "description": "Newsfeed that lists all of the reports and stories that are posted to the app. The reports include a title, brief description, date, and location and can be clicked on to see more information."
+        "title": "Professors create a new class project.",
+        "description": "Title the project and indicate how many students should be in each group.",
+        "image": "/static/projects/zing-feature1.png"
+      },
+      {
+        "title": "Students sign up for the project.",
+        "description": "All multiple-choice questions in 3 minutes so we learn about your study habits and group preferences to match you best.",
+        "image": "/static/projects/zing-feature2.png"
       },
       {
-        "title": "Go Out.",
-        "description": "The map interface that includes a search bar to find locations and reports on campus."
+        "title": "Professors match students into groups with one click.",
+        "description": "Specify the sorting criteria for students and let our algorithm take care of the rest. After, you can filter by year, major, gender, and more to see the distribution. Drag and drop to swap students. It’s as easy as 1, 2, 3.",
+        "image": "/static/projects/zing-feature3.png"
       },
       {
-        "title": "Reach Out.",
-        "description": "The Resources page that lists hotlines available for those in need and organizations and centers to contact on campus."
+        "title": "Find study partners for classes.",
+        "description": "Students who want an LSC study partner fills out survey to request the class(es) they would like support in. The survey also records their study preferences and other key information to ensure a successful partner match.",
```

Comment:
`fill` instead of `fills`

Reply:
and probably "fill out a survey" instead of "fill out survey"